modules/appeals_api/config/schemas/v2/10182.json

```diff
@@ -0,0 +1,150 @@
+// if/when changes are made to this file, please notify VSA claims & appeals team to make corresponding changes in
+// lib/decision_review/schemas/NOD_create_request_body_schema.json
+
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "JSON Schema for VA Form 10182",
+  "$ref": "#/definitions/nodCreateRoot",
+  "definitions": {
+
+    "nonBlankString": {
+      "type": "string",
+      "pattern": "[^ \\f\\n\\r\\t\\v\\u00a0\\u1680\\u2000-\\u200a\\u2028\\u2029\\u202f\\u205f\\u3000\\ufeff]",
+      "$comment": "The pattern used ensures that a string has at least one non-whitespace character. The pattern comes from JavaScript's \\s character class. \"\\s Matches a single white space character, including space, tab, form feed, line feed, and other Unicode spaces. Equivalent to [ \\f\\n\\r\\t\\v\\u00a0\\u1680\\u2000-\\u200a\\u2028\\u2029\\u202f\\u205f\\u3000\\ufeff].\": https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Character_Classes  We are using simple character classes at JSON Schema's recommendation: https://tools.ietf.org/html/draft-handrews-json-schema-validation-01#section-4.3"
+    },
+
+    "date": { "type": "string", "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}$", "maxLength": 10, "minLength": 10 },
+
+    "nodCreateRoot": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "data": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "type": { "type": "string", "enum": ["noticeOfDisagreement"] },
+            "attributes": {
+              "type": "object",
+              "additionalProperties": false,
+              "properties": {
+                "veteran": {
+                  "type": "object",
+                  "additionalProperties": false,
+                  "properties": {
+                    "homeless": { "type": "boolean" },
+                    "address": {
```

Comment:
Is it possible to have this address definition be the same as our other v2 address schemas? If so it would be a good contender for extraction into a shared schema later :)

Reply:
Updated this to match HLR v2's address schema.